### PR TITLE
add int pad support for Pad1D/2D/3D

### PR DESCRIPTION
**Pull request by littletomatodonkey**
### PR types
Function optimization
### PR changes
APIs
### Describe
Add support for `int type pad` in nn.Pad1D/2D/3D.

### CODE REVIEW

File: python/paddle/nn/layer/common.py [hunk 1]
```diff
@@ -915,7 +922,8 @@ class Pad1D(layers.Layer):
     If mode is 'reflect', pad[0] and pad[1] must be no greater than width-1.
 
     Parameters:
-        padding (Tensor | List[int32]): The padding size with data type int32. [len(padding)/2] dimensions
+        padding (Tensor | List[int32], int): The padding size with data type int32. If is int, use the
```
Comment (willthefrog) [inline python/paddle/nn/layer/common.py]: unrelated to current PR but `int32` does not seem correct, by default python 3 int is int64

**Review by willthefrog — APPROVED** [review]
LGTM

File: python/paddle/nn/layer/common.py (hunk 1 above)
Comment (littletomatodonkey) [reply python/paddle/nn/layer/common.py]: done, thx~

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by ceci3 — APPROVED** [review]
LGTM

**Pull request merged**